sandbox/public/legacy.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+    <script src="https://github.com/Adobe-Marketing-Cloud/id-service/releases/download/4.4.1/visitorapi.min.js"></script>
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self' *.demdex.net;
+            script-src 'self' 'unsafe-inline' assets.adobedtm.com cdn.tt.omtrdc.net;
+            style-src 'self' 'unsafe-inline' cdn.tt.omtrdc.net;
+            img-src * data:;
+            connect-src 'self' unifiedjslab.tt.omtrdc.net demdex.net *.demdex.net http://localhost:8080 https://alpha.konductor.adobedc.net https://edgegateway.azurewebsites.net konductor-qe.apps-exp-edge-npe-va6.experience-edge.adobeinternal.net"
+    />
+
+    <title>Mock website hosting Alloy</title>
+    
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+          "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+          "Helvetica Neue", sans-serif;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+      }
+
+      code {
+        font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+          monospace;
+      }
+
+      section {
+        padding: 15px;
+      }
+
+      button {
+        display: block;
+        margin-bottom: 15px;
+      }
+
+      li,
+      button {
+        display: inline-block;
+        margin-right: 20px;
+        font-size: 120%;
+      }
+
+      #root {
+        padding: 10px;
+      }
+    </style>
+  </head>
+  <script> 
+    Visitor.getInstance("53A16ACB5CC1D3760A495C99@AdobeOrg", {});
+  </script>
+  <body>
+    <div>
+      <ul>
+        <li>
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/about">About Us</a>
+        </li>
+        <li>
+          <a href="/orgTwo">Organization Two</a>
+        </li>
+        <li>
+          <a href="/links">Links</a>
+        </li>
+        <li>
+          <a href="/eventMerge">EventMerge</a>
+        </li>
+        <li>
+          <a href="/optIn">OptIn</a>
+        </li>
+        <li>
+          <a href="/largePayload">LargePayload</a>
+        </li>
+        <li>
+          <a href="/DualTag">DualTag</a>
+        </li>
+        <li>
+          <a href="http://localhost:3000/legacy.html">legacy</a>
```

Comment:
You don't need the full domain here right?

Reply:
Your're right. Thanks!